main.cc

```diff
@@ -1024,6 +1024,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             dbcfg.memtable_to_cache_scheduling_group = make_sched_group("memtable_to_cache", "mt2c", 200);
             dbcfg.gossip_scheduling_group = make_sched_group("gossip", "gms", 1000);
             dbcfg.commitlog_scheduling_group = make_sched_group("commitlog", "clog", 1000);
+            dbcfg.schema_commitlog_scheduling_group = make_sched_group("schema_commitlog", "sclg", 1000);
```

Comment:
sclog or schema_clog perhaps for consistency with the prevoius name (stretching my intelligence trying to contribute to this patch to the extreme)

Reply:
I'd love to use "sclog", but there is a hard-coded 4-character limit on the size of the short name of the scheduling group.